tests/test_batch_change.py

```diff
@@ -76,27 +83,35 @@ def test_get_batch_change(mocked_responses, vinyldns_client):
 
     drc = DeleteRecordSetChange(forward_zone.id, forward_zone.name, 'baz', 'baz.bar.com', RecordType.A, 'Complete',
                                 'id2', 'system-message', 'rchangeid2', 'rsid2')
-    bc = BatchChange('user-id', 'user-name', 'batch change test', datetime.utcnow(), [arc, drc],
-                     'bcid', 'owner-group-id')
+
+    bc = BatchChange('user-id', 'user-name', datetime.utcnow(), [arc, drc],
```

Comment:
Can you do a little research on python datetime.  I know there is idiomatic "make sure there is a timezone" somewhere.  We should use the same in our test and code so we can point to it as an example

Reply:
@pauljamescleary if the user passes in the time as something like `datetime(2019, 8, 2, 8, 54, 59, 171208).astimezone()` then it includes their local time and we can change it to utc and then the string format the api is expecting. If they don't include `astimezone()` though we won't have any way of knowing if they are giving us utc time or their local time.


Not sure if it would be better to tell users to use local time and include the `astimezone()` or tell them we expect the time datetime to be in utc.

And either way i'm assuming we'll take datetime and then format it into the string for the API.

Reply:
We can raise a ValueError if the scheduled time does not have a time zone, that way we don't "guess" what the user's intention was